etc/cmake/options.cmake

```diff
@@ -190,8 +190,6 @@ else()
     message(FATAL_ERROR "Invalid max RCP restoration count: ${OT_RCP_RESTORATION_MAX_COUNT}")
 endif()
 
-option(OT_EXCLUDE_TCPLP_LIB "exclude TCPlp library from build")
```

Comment:
Thanks @mariuszpos .

In general, make sense to me to remove this now that we have `OT_TCP`. But can removing this cause issue with projects that integrated OT stack and were using/setting this option?

Suggest either keeping it or adding checks to trigger error if we see it being used?

Reply:
ok, I've added triggering such an error